src/manifest.json

```diff
@@ -1,15 +1,21 @@
 {
-  "name": "RocketChat",
-  "short_name": "RocketChat",
+  "name": "Rocket.Chat",
+  "short_name": "Rocket.Chat",
   "start_url": ".",
   "display": "fullscreen",
   "background_color": "#04436a",
+  "theme_color": "#04436a",
   "description": "Amazing chat app",
   "orientation": "portrait-primary",
   "icons": [
     {
-      "src": "assets/images/rocketchat-bubble.png",
```

Comment:
you must have at least one icon 144X144 dimensions in order the browser will generate the splash screen.
Also, each platform can and does require its own unique file sizes, this allows the user agent (browser) to determine which images it can use, and which ones it ignores. so yes it should be fine (icon generator gave me this dimensions)